components/brave_rewards/browser/rewards_helper.h

```diff
@@ -2,8 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#ifndef BRAVE_BROWSER_PAYMENTS_PAYMENTS_HELPER_
-#define BRAVE_BROWSER_PAYMENTS_PAYMENTS_HELPER_
+#ifndef BRAVE_BROWSER_BRAVE_REWARDS_REWARDS_HELPER_
```

Comment:
Linter will complain here once set. Should be:
`BRAVE_BROWSER_BRAVE_REWARDS_BROWSER_REWARDS_HELPER_`